openverifiablellm/utils.py

```
@@ -8,24 +8,27 @@
 import logging
 import json
 import platform
-from typing import Union, Optional
+from typing import Union, Optional, Dict, Any, List, Tuple
```

Ruff (0.15.2)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)